### ISSUE TITLE
remove Lambda encodings

### ISSUE THREAD

**Issue by asah** [asah/codesmell#1]
Currently, Keras code uses Lambda functions for encoding the characters, which breaks two things:
1. can't run keras-js (in browser prediction): https://github.com/transcranial/keras-js
2. trained models (.hdf5) are specific to the given encoding

The code to replace:

def binarize(x, sz=97):
    import tensorflow as tf2
    return tf2.to_float(tf2.one_hot(x, sz, on_value=1, off_value=0, axis=-1))

def binarize_outshape(in_shape, sz=97):
    return in_shape[0], in_shape[1], sz
...
embedded = Lambda(binarize, output_shape=binarize_outshape)(line_of_code_input)
